microsite/src/main/tut/interop/reactivestreams.md

```diff
@@ -6,23 +6,70 @@ title:  "Reactive Streams"
 
 # {{page.title}}
 
-Checkout `interop-reactiveStreams` module for inter-operation support.
+Checkout the `interop-reactiveStreams` module for inter-operation support.
 
 ### Reactive Streams `Producer` and `Subscriber`
 
 **ZIO** integrates with [Reactive Streams](http://reactivestreams.org) by providing conversions from `zio.stream.Stream` to `org.reactivestreams.Publisher`
-and from `zio.stream.Sink` to `org.reactivestreams.Subscriber` and vice versa.
+and from `zio.stream.Sink` to `org.reactivestreams.Subscriber` and vice versa. Simply import `import scalaz.zio.interop.reactiveStreams._` to make the 
+conversions available.
 
-#### Example
+### Examples
 
+#### Publisher to Stream
 ```scala
```

Comment:
Can we add running examples using `tut`?

Reply:
like so?

Reply:
Yup, thanks. How about running the streams in the examples as well?

Reply:
done. 
For Stream to Publisher we only see the unit result of subscribing the Subscriber to the Publisher. Making more visible here would require a lot of code around the actual business logic, awaiting a Promise, which is completed onComplete from the Subscriber. That would add a lot of noise around the actually pretty simpel code, so I left that out.